website/docs/usage/models.md

```diff
@@ -59,6 +38,31 @@ your data.
 > $ pip install -U %%SPACY_PKG_NAME[lookups]%%SPACY_PKG_FLAGS
 > ```
 
+If a trained pipeline is available for a language, you can download it using the
+[`spacy download`](/api/cli#download) command as shown above. In order to use
+languages that don't yet come with a trained pipeline, you have to import them
+directly, or use [`spacy.blank`](/api/top-level#spacy.blank):
+
+```python
+from spacy.lang.fi import Finnish
+nlp = Finnish()  # use directly
+nlp = spacy.blank("fi")  # blank instance
+```
```

Comment:
I think this example will be less confusing if we switch to something that currently doesn't have a pipeline, so not Finnish.

Reply:
Changed to Ukrainian.

Reply:
Sorry, there's no way for you to know, but `Ukrainian` may be added soon-ish. How about `Yoruba`?